windpowerlib/power_output.py

```diff
@@ -247,24 +247,12 @@ def power_curve_density_correction(
     else:
         panda_series = False
 
-    power_output = [
-        (
-            np.interp(
-                wind_speed[i],
-                power_curve_wind_speeds
-                * (1.225 / density[i])
-                ** (
-                    np.interp(
-                        power_curve_wind_speeds, [7.5, 12.5], [1 / 3, 2 / 3]
-                    )
-                ),
-                power_curve_values,
-                left=0,
-                right=0,
-            )
-        )
-        for i in range(len(wind_speed))
-    ]
+    power_output = _get_power_output(
+        wind_speed,
+        power_curve_wind_speeds.to_numpy(),
+        density.to_numpy(),
+        power_curve_values.to_numpy(),
+    )
```

Comment:
The problem is that we also allow numpy-arrays as an input, in that case `to_numpy` will not work.

I think `numpy.array()` should work for Series and Arrays.

```diff
- power_curve_wind_speeds.to_numpy(),
+ numpy.array(power_curve_wind_speeds)
```

Reply:
Ah thanks for the comments! I couldn't see what was failing with Travis, only that an error occured.

Hopefully the new commit fixes it